pkg/cmd/release/upload/upload.go

```diff
@@ -93,9 +94,11 @@ func uploadRun(opts *UploadOptions) error {
 
 	var existingNames []string
 	for _, a := range opts.Assets {
+		sanitizedFileName := sanitizeFileName(a.Name)
 		for _, ea := range release.Assets {
-			if ea.Name == a.Name {
+			if ea.Name == sanitizedFileName {
 				a.ExistingURL = ea.APIURL
+				a.Name = ea.Name
```

Comment:
Is it necessary to override the `Name` field here? To my knowledge, only ExistingURL field is used when deleting assets. For other code that might operate on `opts.Assets`, it might be good if `a.Name` remained unchanged.

Reply:
good call, for some reason when I was testing this earlier, I thought it needed this, but I removed it and it seems to work correctly now